Packs/Base/Scripts/ValidateContent/ValidateContent.py

```diff
@@ -188,15 +189,16 @@ def run_validate(file_path: str, json_output_file: str) -> None:
 
 def run_lint(file_path: str, json_output_file: str) -> None:
     lint_log_dir = os.path.dirname(json_output_file)
-    logging_setup(verbose=3, quiet=False, log_path=lint_log_dir)
+    logging_setup(3)
```

Comment:
@tkatzir is this still supported with the [new implementation](https://github.com/demisto/demisto-sdk/blob/7ff03545c6338ffb9baa3083375a9293d65891e9/demisto_sdk/commands/common/logger.py#L102)? 
I recommend explicitly passing the arguments instead of `(3)`

Reply:
fixed